### PR TITLE
Dragging - Add more objects configs

### PR DESCRIPTION
**Pull request by Dystopian**
**When merged this pull request will:**
- add dragging configurations for frequent terrain objects (~requires #8103~);
- add default `Position` for missing config values;
- cleanup dragging config a little;
- change default draggable position from `[0,0,0]` (useless because it's inside player) to `[0,1.5,0]` (suits many objects);
- update documentation according to changes.

Use this code to create new objects for testing:
```
{createVehicle [_x, player]} forEach ["Land_BarrelSand_F","Land_BarrelWater_F","Land_Basket_F","Land_Bucket_F","Land_CampingChair_V1_folded_F","Land_CampingTable_F","Land_CampingTable_small_F","Land_CanisterPlastic_F","Land_CratesPlastic_F","Land_GarbageBarrel_01_english_F","Land_GarbageContainer_closed_F","Land_GarbageContainer_open_F","Land_MetalBarrel_empty_F","Land_MetalBarrel_F","Land_Pallet_F","Land_Pallet_vertical_F","Land_Pallets_F","Land_Sun_chair_F","Land_TablePlastic_01_F","Land_Tyre_F","Land_WheelCart_F","Land_WoodenCart_F","Land_WoodenTable_large_F","Land_WorkStand_F"];
```

### CODE REVIEW

File: docs/wiki/framework/dragging-framework.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ class CfgVehicles {
     class MyVehicle {
         // Dragging
         ace_dragging_canDrag = 1;  // Can be dragged (0-no, 1-yes)
-        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo)
-        ace_dragging_dragDirection = 0;  // Model direction while dragging (same as setDir after attachTo)
+        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (optional; default: [0,1.5,0])
```
Comment (jonpas) [inline docs/wiki/framework/dragging-framework.md:22]:
```suggestion
        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (optional; default: [0, 1.5, 0])
```

File: docs/wiki/framework/dragging-framework.md [hunk 2]
```diff
@@ -19,13 +19,13 @@ class CfgVehicles {
...
+        ace_dragging_dragDirection = 0;  // Model direction while dragging (same as setDir after attachTo) (optional; default: 0)
 
         // Carrying
         ace_dragging_canCarry = 1;  // Can be carried (0-no, 1-yes)
-        ace_dragging_carryPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo)
-        ace_dragging_carryDirection = 0;  // Model direction while dragging (same as setDir after attachTo)
+        ace_dragging_carryPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (optional; default: [0,1,1])
```
Comment (jonpas) [inline docs/wiki/framework/dragging-framework.md:27]:
```suggestion
        ace_dragging_carryPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (optional; default: [0, 1, 1])
```

File: docs/wiki/framework/dragging-framework.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ You will **not** be able to carry / drag objects that are too heavy, the mass is
 ---| --------- | ---- | ------------------------
 0  | Any object | Object | Required
 1  | Enable dragging, true to enable, false to disable | Boolean | Required
-2  | Position to offset the object from player | Array | Optional (default: `[0, 0, 0]`)
+2  | Position to offset the object from player | Array | Optional (default: `[0,1.5,0]`)
```
Comment (jonpas) [inline docs/wiki/framework/dragging-framework.md:51]:
```suggestion
2  | Position to offset the object from player | Array | Optional (default: `[0, 1.5, 0]`)
```

File: addons/dragging/functions/fnc_setDraggable.sqf [hunk 4]
```diff
@@ -23,7 +23,7 @@
 params ["_object", "_enableDrag", "_position", "_direction", ["_ignoreWeightDrag", false, [false]]];
 
 if (isNil "_position") then {
-    _position = _object getVariable [QGVAR(dragPosition), [0,0,0]];
+    _position = _object getVariable [QGVAR(dragPosition), [0,1.5,0]];
```
Comment (jonpas) [inline addons/dragging/functions/fnc_setDraggable.sqf:26]:
```suggestion
    _position = _object getVariable [QGVAR(dragPosition), [0, 1.5, 0]];
```

File: addons/dragging/functions/fnc_setDraggable.sqf [hunk 5]
```diff
@@ -6,7 +6,7 @@
  * Arguments:
  * 0: Any object <OBJECT>
  * 1: true to enable dragging, false to disable <BOOL>
- * 2: Position offset for attachTo command (optional; default: [0,0,0])<ARRAY>
+ * 2: Position offset for attachTo command (optional; default: [0,1.5,0]) <ARRAY>
```
Comment (jonpas) [inline addons/dragging/functions/fnc_setDraggable.sqf:9]:
```suggestion
 * 2: Position offset for attachTo command (optional; default: [0, 1.5, 0]) <ARRAY>
```

File: addons/dragging/functions/fnc_initObject.sqf [hunk 6]
```diff
@@ -20,14 +20,14 @@ params ["_object"];
...
-    private _position = getArray (_config >> QGVAR(dragPosition));
+    private _position = [_config >> QGVAR(dragPosition), "ARRAY", [0,1.5,0]] call CBA_fnc_getConfigEntry;
     private _direction = getNumber (_config >> QGVAR(dragDirection));
 
     [_object, true, _position, _direction] call FUNC(setDraggable);
 };
 
 if (getNumber (_config >> QGVAR(canCarry)) == 1) then {
-    private _position = getArray (_config >> QGVAR(carryPosition));
+    private _position = [_config >> QGVAR(carryPosition), "ARRAY", [0,1,1]] call CBA_fnc_getConfigEntry;
```
Comment (jonpas) [inline addons/dragging/functions/fnc_initObject.sqf:30]:
```suggestion
    private _position = [_config >> QGVAR(carryPosition), "ARRAY", [0, 1, 1]] call CBA_fnc_getConfigEntry;
```

File: addons/dragging/functions/fnc_initObject.sqf [hunk 7]
```diff
@@ -20,14 +20,14 @@ params ["_object"];
 private _config = configFile >> "CfgVehicles" >> typeOf _object;
 
 if (getNumber (_config >> QGVAR(canDrag)) == 1) then {
-    private _position = getArray (_config >> QGVAR(dragPosition));
+    private _position = [_config >> QGVAR(dragPosition), "ARRAY", [0,1.5,0]] call CBA_fnc_getConfigEntry;
```
Comment (jonpas) [inline addons/dragging/functions/fnc_initObject.sqf:23]:
```suggestion
    private _position = [_config >> QGVAR(dragPosition), "ARRAY", [0, 1.5, 0]] call CBA_fnc_getConfigEntry;
```
Readability.

**Review by jonpas — CHANGES_REQUESTED** [review]
I think we can merge this right away if you split off terrain replacement config into that other existing PR.

**Comment by Dystopian** [comment]
I removed replacing config so this PR should be merged before #8103, then I re-add changes there.

I also added test code to description to create all new objects.

**Review by jonpas — APPROVED** [review]

File: docs/wiki/framework/dragging-framework.md [hunk 8]
```diff
@@ -19,13 +19,13 @@ class CfgVehicles {
...
-        ace_dragging_dragDirection = 0;  // Model direction while dragging (same as setDir after attachTo)
+        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (optional; default: [0, 1.5, 0])
+        ace_dragging_dragDirection = 0;  // Model direction while dragging (same as setDir after attachTo) (optional; default: 0)
 
         // Carrying
         ace_dragging_canCarry = 1;  // Can be carried (0-no, 1-yes)
-        ace_dragging_carryPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo)
-        ace_dragging_carryDirection = 0;  // Model direction while dragging (same as setDir after attachTo)
+        ace_dragging_carryPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (optional; default: [0, 1, 1])
+        ace_dragging_carryDirection = 0;  // Model direction while dragging (same as setDir after attachTo) (optional; default: 0)
```
Comment (jonpas) [inline docs/wiki/framework/dragging-framework.md:28]:
```suggestion
        ace_dragging_carryDirection = 0;  // Model direction while dragging (same as setDir after attachTo) (default: 0)
```

File: docs/wiki/framework/dragging-framework.md [hunk 9]
```diff
@@ -19,13 +19,13 @@ class CfgVehicles {
...
-        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo)
-        ace_dragging_dragDirection = 0;  // Model direction while dragging (same as setDir after attachTo)
+        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (optional; default: [0, 1.5, 0])
+        ace_dragging_dragDirection = 0;  // Model direction while dragging (same as setDir after attachTo) (optional; default: 0)
 
         // Carrying
         ace_dragging_canCarry = 1;  // Can be carried (0-no, 1-yes)
-        ace_dragging_carryPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo)
-        ace_dragging_carryDirection = 0;  // Model direction while dragging (same as setDir after attachTo)
+        ace_dragging_carryPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (optional; default: [0, 1, 1])
```
Comment (jonpas) [inline docs/wiki/framework/dragging-framework.md:27]:
```suggestion
        ace_dragging_carryPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (default: [0, 1, 1])
```

File: docs/wiki/framework/dragging-framework.md [hunk 10]
```diff
@@ -19,13 +19,13 @@ class CfgVehicles {
     class MyVehicle {
         // Dragging
         ace_dragging_canDrag = 1;  // Can be dragged (0-no, 1-yes)
-        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo)
-        ace_dragging_dragDirection = 0;  // Model direction while dragging (same as setDir after attachTo)
+        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (optional; default: [0, 1.5, 0])
+        ace_dragging_dragDirection = 0;  // Model direction while dragging (same as setDir after attachTo) (optional; default: 0)
```
Comment (jonpas) [inline docs/wiki/framework/dragging-framework.md:23]:
```suggestion
        ace_dragging_dragDirection = 0;  // Model direction while dragging (same as setDir after attachTo) (default: 0)
```

File: docs/wiki/framework/dragging-framework.md [hunk 11]
```diff
@@ -19,13 +19,13 @@ class CfgVehicles {
     class MyVehicle {
         // Dragging
         ace_dragging_canDrag = 1;  // Can be dragged (0-no, 1-yes)
-        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo)
-        ace_dragging_dragDirection = 0;  // Model direction while dragging (same as setDir after attachTo)
+        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (optional; default: [0, 1.5, 0])
```
Comment (jonpas) [inline docs/wiki/framework/dragging-framework.md:22]:
```suggestion
        ace_dragging_dragPosition[] = {0, 1.2, 0};  // Offset of the model from the body while dragging (same as attachTo) (default: [0, 1.5, 0])
```

**Review by jonpas — APPROVED** [review]

**Pull request merged**